tensorflow/stream_executor/rocm/rocm_blas.cc

```diff
@@ -1877,37 +1880,70 @@ port::Status ReorganizeMemory(Stream* stream,
   size_t matrix_byte_size = batch_stride * sizeof(MAPPED_T);  
   uint64_t cur_stride_size = matrix_byte_size;
 
+  struct write_op
+  {
+    char* src_ptr;
+    char* dst_ptr;
+    uint64_t size;
+  };
+  std::vector<write_op> write_ops;
+
   for (int i = 1; i < batch_count; ++i) {
     if (reinterpret_cast<char*>(raw_ptrs[i]) == src_ptr + cur_stride_size) {
       cur_stride_size += matrix_byte_size;
     } else {
-      DeviceMemoryBase src_mem = DeviceMemoryBase(src_ptr, cur_stride_size);
-      DeviceMemoryBase target_mem = DeviceMemoryBase(dst_ptr, cur_stride_size);
+      write_ops.push_back(write_op{src_ptr,dst_ptr,cur_stride_size});
```

Comment:
Nit: Formatting seems off in the change. Could you install `clang-format` and do `git clang-format`, which will only reformat the changes you made to the code base.